cinema_service/urls.py

```diff
@@ -2,10 +2,22 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularSwaggerView,
+    SpectacularRedocView
+)
```

Comment:
The import of views from `drf_spectacular` is correct and allows for API schema generation and documentation using Swagger and Redoc. Ensure that the schema is kept up-to-date with any changes to the API.